microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -220,6 +231,22 @@ def enable_uio_hv_generic_for_nic(node: Node, nic: NicInfo) -> None:
     echo = node.tools[Echo]
     lsmod = node.tools[Lsmod]
     modprobe = node.tools[Modprobe]
+    kconfig = node.tools[KernelConfig]
+    uname = node.tools[Uname]
+
+    # check if kernel config for Hyper-V VMBus is enabled
+    config = "CONFIG_UIO_HV_GENERIC"
+    if not kconfig.is_enabled(config):
+        kversion = uname.get_linux_information().kernel_version_raw
+        if kversion < "4.10":
+            raise SkippedException(
```

Comment:
```uname.get_linux_information().kernel_version_raw``` contains str like 4.9.0-16-amd64, it can't be used for comparison.
Change code like below, it works.
```
        kversion = uname.get_linux_information().kernel_version
        if kversion < "4.10.0":
```